packages/preset-create-react-app/src/helpers/processCraConfig.ts

```diff
@@ -45,7 +45,7 @@ export const processCraConfig = (
     if (testMatch(rule, '.jsx')) {
       const newRule = {
         ...rule,
-        include: [include as string, configDir],
+        include: [include as string, configDir].filter(Boolean),
```

Comment:
```suggestion
        include: include ? [...(Array.isArray(inluclude) ? include : [include]), configDir],
```

wouldn't this need to not set any `include` if there was not `include` specified in the original rule? Otherwise the rule will only be applied to the `configDir` but nothing else. But with cra the rule will be applied to everything, except of what is mentioned in `exclude`